src/components/dashboard/list/TokenListItem.tsx

```diff
@@ -1,20 +1,15 @@
 import { type Token } from "~tokens/token";
 import { Reorder, useDragControls } from "framer-motion";
-import { useMemo, useState } from "react";
+import { useMemo } from "react";
 import { ListItem } from "@arconnect/components-rebrand";
 import { formatAddress } from "~utils/format";
-import { useTheme } from "~utils/theme";
 import styled from "styled-components";
-import { FULL_HISTORY, useGateway } from "~gateways/wayfinder";
-import { concatGatewayURL } from "~gateways/utils";
-import aoLogo from "url:/assets/ecosystem/ao-logo.svg";
-import arLogo from "url:/assets/ecosystem/ar-logo.svg";
-import arLogoDark from "url:/assets/ar/logo_dark.png";
-import { getUserAvatar } from "~lib/avatar";
 import { useLocation } from "~wallets/router/router.utils";
-import CommonImage from "~components/common/Image";
 import { AR_PROCESS_ID } from "~tokens/aoTokens/ao";
-import { useAsyncEffect } from "~utils/react/useAsyncEffect";
+import { TokenLogo } from "~components/popup/TokenLogo";
+
+import aoLogo from "url:/assets/ecosystem/ao-logo.svg";
+import arLogo from "url:/assets/ecosystem/ar-logo.svg";
```

Comment:
Always shown in the same color, so importing it directly rather than using `TokenLogo`.